pages/firststeps.md

```diff
@@ -1,12 +1,29 @@
 # First Steps
 
-Welcome to the first steps for becoming an OLE Virtual Intern! We treat these first steps as a vetting process to prove that you can follow simple steps before you can progress to working on harder projects in bigger teams. Because these steps are simple, we expect high quality work, which may take a longer time. We want to see that you are capable of using or learning how to use these tools (writing good GitHub issues, creating pull requests, navigating the BeLL, etc.). These steps may seem easy, but we want you to impress us with good GitHub etiquette and quality Markdown. The bare minimum would be to just passively follow the steps; you should do further reading about the tools/languages we use so you can further your understanding and relieve confusion if you're unclear about how something works. 
+# The Big Picture
 
-We created a [FAQ page](faq.md) where you can find the answers to some commonly asked questions, and also some very useful links and videos tutorials that will help you become familiar with the tools/languages we use. For anything that is not in the FAQ page, Google and Stack Exchange are your friends :)
+Welcome to the first steps for becoming an OLE Virtual Intern! We treat these first steps as a vetting process to prove that you can follow simple instructions before you can progress to working on harder projects in bigger teams. Consider them to be the interview for the internship. 
 
-We also want you to keep us updated a relatively good amount in the [Gitter chat](https://gitter.im/open-learning-exchange/chat) as you complete these steps. Generally, you should comment in the Gitter chat when you make an issue or pull request, and then link to it. It's also good form to mention someone in Gitter when you comment on their issue/pull request, and of course link to it. For example: "Hey @dogi, just commented on your issue: https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/15"
+If you are deemed fit for the internship after completing the steps, you will be officially invited to join the OLE interns team! We’ll add you to the interns Gitter chatroom and assign you to a specific team to work on developing and improving OLE’s BeLL (Basic e-Learning Library) software. The BeLL is a virtual library that is deployed internationally to children in countries that typically do not have access to educational resources.
+
+Once accepted, you and your team will work on an assignment, and we’ll switch up the assignments each week. As part of this internship, you will be working with software and languages including Git, GitHub, Gitter, Markdown, Vagrant, VirtualBox, Command Line/Terminal, Command Line/Terminal Scripts, Vim, CouchDB, Docker, HTML5, Javascript, and Node.js.
```

Comment:
@waryhermit Sounds great! I don't really know anything about that stuff though since I'm not a part of it, so if you could provide some more insight than we can add it ;)